test/com/google/enterprise/adaptor/database/JdbcFixture.java

```diff
@@ -48,9 +54,34 @@ public static Connection getConnection() {
   }
 
   public static void dropAllObjects() throws SQLException {
+    try {
+      for (AutoCloseable object : openObjects) {
+        object.close();
+      }
+    } catch (Exception e) {
```

Comment:
Yes. I thought about it, but the chances of a problem are rare, and likely point out flaws in the test.